src/mango/test/01-index-crud-test.py

```diff
@@ -68,17 +68,17 @@ def test_bad_types(self):
             else:
                 raise AssertionError("bad create index")
 
-    def test_bad_names(self):
-        bad_names = [True, False, 1.5, {"foo": "bar"}, [None, False]]
-        for bn in bad_names:
+    def test_bad_names_ddocs(self):
+        bad_args = ["", True, False, 1.5, {"foo": "bar"}, [None, False]]
```

Comment:
this only tests for `"name": ""` ? Can we add a testcase where `"ddoc": ""` can it receives the same error? The original bug has `"ddoc" : ""`

Reply:
Added another test case [test_bad_ddocs()](https://github.com/apache/couchdb/pull/3990/files#diff-243cff897c08c3a4b060fed2dc84a22d35740500fbf075e6209ba5013f0fd7faR81), thanks for the review.